thunder/tests/opinfos.py

```diff
@@ -5873,6 +5873,37 @@ def tensor_constructor_error_generator(op, device, **kwargs):
 linear_algebra_ops = []
 
 
+def normalize_sample_generator(op, device, dtype, requires_grad, **kwargs):
+    make = partial(make_tensor, device=device, dtype=dtype, requires_grad=requires_grad)
+    # input shape
+    cases = (
+        (4, 4),
+        (1024, 1024),
```

Comment:
This is a great case, but probably too big for a "sample" (we have a separate concept of "reference inputs" for larger inputs, for example:

https://github.com/Lightning-AI/lightning-thunder/blob/067f15aae47ad71229732ca6c35a5d190135e48c/thunder/tests/opinfos.py#L7017

)

The "samples" are typically smaller inputs so they can be used in more tests without taking too much CI time. Maybe `(32, 32)` would be OK?

Reply:
Yeah absolutely! Thanks for explaining the differences between reference inputs and samples.